packages/syntax-core/src/RichSelect/RichSelectRadioButton.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { forwardRef, type ReactElement } from "react";
+import RichSelectItem, { type RichSelectItemProps } from "./RichSelectItem";
+import RadioButton from "../RadioButton/RadioButton";
+
+const NOOP = () => undefined;
+
+type RichSelectRadioButtonProps = RichSelectItemProps & {
+  name: string;
+  size?: "sm" | "md";
```

Comment:
remove size for now?

Reply:
left the prop so that `size` can stil be set for the `RichSelectRadioButton`.   The design we have that needs richselectlist uses `sm` radiobuttons, while the normal default is `md`